.husky/pre-commit

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+npx eslint ui/ --fix && npx pretty-quick --staged ui/ --write --loglevel warn
```

Comment:
I'm pretty sure we don't want eslint at this point, because:
a) it will fail on non-staged code
b) it (afaik) won't stage any changes it makes
c) may make prettier fail out when it doesn't need to if it's changes result in a partially staged file (assuming I'm reading the `pretty-quick` docs correctly)

```suggestion
npx pretty-quick --staged ui/ --write --loglevel warn
```